src/lib/stable/ic_apis/call.ts

```diff
@@ -23,12 +24,15 @@ export async function call(
         const globalRejectId = `_reject_${promiseId}`;
 
         const returnIdl = options?.returnIdl;
+        const raw = options?.raw;
 
         // TODO perhaps we should be more robust
         // TODO for example, we can keep the time with these
         // TODO if they are over a certain amount old we can delete them
         globalThis._azleResolveIds[globalResolveId] = (result: ArrayBuffer) => {
-            if (returnIdl === undefined) {
+            if (raw !== undefined) {
+                resolve(new Uint8Array(result));
+            } else if (returnIdl === undefined) {
```

Comment:
I'm just thinking about this a little bit... in the case where the developer leaves off the returnIDL his function is going to be returning undefined. 

But if he had put the wrong IDL then he would have gotten an error that is something like "failed to serialize <value> to <type>. 

I'm wondering if that would be a more instructive error to help them know that they are missing the return IDL instead of just returning undefined.

Reply:
This seems like a good idea, let's implement it later and make sure to test it: https://github.com/demergent-labs/azle/issues/1864